scripts/zones/Ranguemont_Pass/mobs/Taisai.lua

```diff
@@ -3,29 +3,34 @@
 -- MOB:  Taisai
 -----------------------------------
 
+require("scripts/globals/status");
+local MobIDs = require("scripts/zones/Ranguemont_Pass/MobIDs");
+
 function onMobRoam(mob)
 
-    local Taisaijin = 17457216;
-    local Taisaijin_PH = 0;
-    local Taisaijin_PH_Table =
-    {
-        17457213,
-        17457214,
-        17457215
-    };
-    local Taisaijin_ToD = GetMobByID(Taisaijin):getLocalVar("ToD");
-
-    if (Taisaijin_ToD <= os.time()) then
-        Taisaijin_PH = math.random((0), (#Taisaijin_PH_Table));
-        if (Taisaijin_PH_Table[Taisaijin_PH] ~= nil) then
-            if (GetMobAction(Taisaijin) == 0) then
-                SetServerVariable("Taisaijin_PH", Taisaijin_PH_Table[Taisaijin_PH]);
-                DeterMob(Taisaijin_PH_Table[Taisaijin_PH], true);
-                DeterMob(Taisaijin, false);
-                DespawnMob(Taisaijin_PH_Table[Taisaijin_PH]);
-                SpawnMob(Taisaijin, "", 0);
-            end
+    local tts = GetServerVariable("Taisaijin_TTS");
+    if (tts > 0 and tts <= os.time() and GetMobAction(MobIDs.Taisaijin) == ACTION_NONE) then
```

Comment:
Sorry I didn't see the whole comment before, I think I read the truncated email.

I have a PR from about a year ago where I did just that.   I wouldn't mind taking my time and doing it little by little though.  I ran across several bugs with global leaks in a few zones where values were getting stomped on.

If you do adopt a standard for this, I highly recommend the one here, to avoid accidental variable corruption.